odoo/addons/base/tests/test_ir_asset.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.modules.module import get_module_resource
+from odoo.tests.common import TransactionCase
+from odoo.tools import convert_file
+
+
+class TestAsset(TransactionCase):
+
+    def test_00_asset_tag(self):
+        """
+        Verify that assets defined with the <asset> tag are properly imported.
+        """
+        # Load new records
+        convert_file(self.cr, 'base',
+            get_module_resource('base', 'tests', 'asset_tag.xml'),
+            {}, 'init', False, 'test')
```

Comment:
![image](https://user-images.githubusercontent.com/30048408/210999827-1d909a3d-71c9-4697-820e-3b18aa036d98.png)
```suggestion
        convert_file(
            self.cr, 'base',
            get_module_resource('base', 'tests', 'asset_tag.xml'),
            {}, 'init', False, 'test'
        )
```